src/gui_tool/gui/sp_gui.py

```diff
@@ -29,23 +39,22 @@ class SPGUIManager(VideoPlayerGUIManager):
     LOG_LINE_HEIGHT = 17
     LOG_LINE_MARGIN = 2
     LOG_START_X = 250
-    IMG_STARTING_Y = LOG_LINE_HEIGHT * LOG_LINES + LOG_LINE_MARGIN * (LOG_LINES + 1) + 3
+    IMG_STARTING_Y = LOG_LINE_HEIGHT * LOG_LINES + LOG_LINE_MARGIN * (
+        LOG_LINES + 1) + 3
 
     def __init__(self, context: BBContext):
-        super(SPGUIManager, self).__init__(context,
-           [
-               SPMode(self, context)
-           ]
-        )
+        super(SPGUIManager, self).__init__(context, [SPMode(self, context)])
 
     def get_return_fields(self) -> (List[Section], List[List]):
-        sections = self.mode_handlers[0].sm.get_all_sections()
+        sections = [
```

Comment:
while this is not actually a necessary part of this PR, I think this was a bug from before since every time this function is called, we would add to the start and end of each section (thus getting the section actually changed the sections data lol)